<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>GameScreen</class>
 <widget class="QDialog" name="GameScreen">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>470</width>
    <height>405</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QGroupBox" name="gameplayscreen">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>451</width>
     <height>381</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>-1</pointsize>
     <weight>75</weight>
     <italic>false</italic>
     <bold>true</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">QGroupBox {
    font: bold;
	font-size:16px;
    border: 1px solid silver;
    border-radius: 12px;
    margin-top: 8px;
    padding: 15px 5px 10px 5px;
background-color: rgb(245, 245, 245);
}

QGroupBox::title {
    subcontrol-origin: margin;
    left: 7px;
    padding: 0px 5px 0px 5px;
}

QLabel{
	background-color: none;
}</string>
   </property>
   <property name="title">
    <string>Player</string>
   </property>
   <widget class="QPushButton" name="player_order">
    <property name="geometry">
     <rect>
      <x>210</x>
      <y>120</y>
      <width>142</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>DejaVu Sans</family>
      <pointsize>-1</pointsize>
      <weight>75</weight>
      <italic>false</italic>
      <bold>true</bold>
      <underline>false</underline>
      <strikeout>false</strikeout>
     </font>
    </property>
    <property name="cursor">
     <cursorShape>PointingHandCursor</cursorShape>
    </property>
    <property name="styleSheet">
     <string notr="true">#player_order {
    background-color: rgb(106,208,116);
background-color: qlineargradient(spread:pad, x1:0, y1:0.005, x2:1, y2:1, stop:0 rgba(66, 163, 167, 255), stop:1 rgba(76, 92, 171, 255));
    border-style: outset;
    border-width: 2px;
    border-radius: 10px;
    border-color: beige;
    font: bold 18px;
    min-width: 6em;
    padding: 6px;
	color: white;
}

#player_order:hover { 
background-color: rgba(76, 92, 171, 255); 
color: white; 
border: 2px solid rgba(66, 163, 167, 255);}


#player_order:disabled {
	background-color: #add8e6;
	color: black;
}</string>
    </property>
    <property name="text">
     <string> Order</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="order_amount">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>120</y>
      <width>81</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>-1</pointsize>
      <weight>75</weight>
      <italic>false</italic>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">#order_amount{
    border-width: 2px;
    border-radius: 6px;
    border-color: beige;
    font: bold 18px;
    padding: 2px;
	color: black;
}</string>
    </property>
   </widget>
   <widget class="QLabel" name="totalCost_l">
    <property name="geometry">
     <rect>
      <x>280</x>
      <y>340</y>
      <width>91</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>13</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Total Cost:</string>
    </property>
   </widget>
   <widget class="QLabel" name="weekCost_l">
    <property name="enabled">
     <bool>true</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>340</y>
      <width>141</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>13</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Last Week's Cost:</string>
    </property>
   </widget>
   <widget class="QLabel" name="incomingOrder">
    <property name="geometry">
     <rect>
      <x>240</x>
      <y>70</y>
      <width>51</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>13</pointsize>
     </font>
    </property>
    <property name="layoutDirection">
     <enum>Qt::RightToLeft</enum>
    </property>
    <property name="text">
     <string>NA</string>
    </property>
   </widget>
   <widget class="QLabel" name="outgoing_4">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>70</y>
      <width>141</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>13</pointsize>
     </font>
    </property>
    <property name="layoutDirection">
     <enum>Qt::RightToLeft</enum>
    </property>
    <property name="text">
     <string>Incoming Order:</string>
    </property>
   </widget>
   <widget class="QLabel" name="weekCost">
    <property name="enabled">
     <bool>true</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>170</x>
      <y>340</y>
      <width>31</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>13</pointsize>
     </font>
    </property>
    <property name="text">
     <string>0</string>
    </property>
   </widget>
   <widget class="QLabel" name="totalCost">
    <property name="geometry">
     <rect>
      <x>390</x>
      <y>340</y>
      <width>31</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>13</pointsize>
     </font>
    </property>
    <property name="layoutDirection">
     <enum>Qt::RightToLeft</enum>
    </property>
    <property name="styleSheet">
     <string notr="true"/>
    </property>
    <property name="text">
     <string>0</string>
    </property>
   </widget>
   <widget class="QLabel" name="totalCost_l_2">
    <property name="geometry">
     <rect>
      <x>310</x>
      <y>20</y>
      <width>91</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true"/>
    </property>
    <property name="text">
     <string>Game Week:</string>
    </property>
   </widget>
   <widget class="QLabel" name="week">
    <property name="geometry">
     <rect>
      <x>410</x>
      <y>20</y>
      <width>21</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="layoutDirection">
     <enum>Qt::RightToLeft</enum>
    </property>
    <property name="text">
     <string>0</string>
    </property>
   </widget>
   <widget class="QGroupBox" name="currentBox">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>170</y>
      <width>411</width>
      <height>161</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>-1</pointsize>
      <weight>75</weight>
      <italic>false</italic>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QLabel{
	background-color: none;
}</string>
    </property>
    <property name="title">
     <string>Player Statistics Last Week</string>
    </property>
    <widget class="QLabel" name="incomingShipment">
     <property name="geometry">
      <rect>
       <x>150</x>
       <y>120</y>
       <width>39</width>
       <height>20</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>13</pointsize>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">QLabel{
	background-color: none;
}</string>
     </property>
     <property name="text">
      <string>NA</string>
     </property>
    </widget>
    <widget class="QLabel" name="incoming_2">
     <property name="geometry">
      <rect>
       <x>21</x>
       <y>120</y>
       <width>123</width>
       <height>20</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>13</pointsize>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">QLabel{
	background-color: none;
}</string>
     </property>
     <property name="text">
      <string>Received beers:</string>
     </property>
    </widget>
    <widget class="QLabel" name="outgoingShipment">
     <property name="geometry">
      <rect>
       <x>360</x>
       <y>120</y>
       <width>37</width>
       <height>20</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>13</pointsize>
      </font>
     </property>
     <property name="layoutDirection">
      <enum>Qt::RightToLeft</enum>
     </property>
     <property name="text">
      <string>NA</string>
     </property>
    </widget>
    <widget class="QLabel" name="outgoing_2">
     <property name="geometry">
      <rect>
       <x>228</x>
       <y>120</y>
       <width>126</width>
       <height>20</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>13</pointsize>
      </font>
     </property>
     <property name="layoutDirection">
      <enum>Qt::RightToLeft</enum>
     </property>
     <property name="text">
      <string>Delivered Beers:</string>
     </property>
    </widget>
    <widget class="QLabel" name="inventory_2">
     <property name="geometry">
      <rect>
       <x>28</x>
       <y>40</y>
       <width>135</width>
       <height>23</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>13</pointsize>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">QLabel{
	background-color: none;
}</string>
     </property>
     <property name="text">
      <string>Inventory</string>
     </property>
    </widget>
    <widget class="QLabel" name="inventory">
     <property name="geometry">
      <rect>
       <x>230</x>
       <y>40</y>
       <width>95</width>
       <height>23</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>-1</pointsize>
       <weight>75</weight>
       <italic>false</italic>
       <bold>true</bold>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">#inventory{
	background-color:red;
    border-width: 2px;
    border-radius: 5px;
    border-color: beige;
    font: bold 16px;
	max-width: 5em;
	color: white;
}</string>
     </property>
     <property name="text">
      <string>1</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
    <widget class="QLabel" name="backLog_2">
     <property name="geometry">
      <rect>
       <x>28</x>
       <y>70</y>
       <width>135</width>
       <height>22</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>13</pointsize>
      </font>
     </property>
     <property name="layoutDirection">
      <enum>Qt::RightToLeft</enum>
     </property>
     <property name="styleSheet">
      <string notr="true">QLabel{
	background-color: none;
}</string>
     </property>
     <property name="text">
      <string>BackLog</string>
     </property>
    </widget>
    <widget class="QLabel" name="backLog">
     <property name="geometry">
      <rect>
       <x>230</x>
       <y>70</y>
       <width>95</width>
       <height>22</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>-1</pointsize>
       <weight>75</weight>
       <italic>false</italic>
       <bold>true</bold>
      </font>
     </property>
     <property name="layoutDirection">
      <enum>Qt::RightToLeft</enum>
     </property>
     <property name="styleSheet">
      <string notr="true">#backLog{
	background-color:red;
    border-width: 2px;
    border-radius: 5px;
    border-color: beige;
    font: bold 16px;
	max-width: 5em;
	color: white;
}</string>
     </property>
     <property name="text">
      <string>1</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
